.github/workflows/autofix.yml

```
@@ -34,13 +30,29 @@ jobs:
-      - name: Fix linting issues
-        # Skip manypkg fix for Version Packages PRs to avoid version adjustments
+      # Note: We only skip manypkg for Changesets-driven version bumps.
+      # This avoids touching versions that haven't been published yet,
+      # while still letting autofix handle normal PRs and pushes.
+      - name: Check if manypkg should be skipped
+        id: skip-manypkg
+        # Skip manypkg fix for Version Packages PRs and commits to avoid version adjustments
         # on unpublished versions from changesets
-        env:
-          HEAD_REF: ${{ github.event.pull_request.head.ref }}
         run: |
```

actionlint (1.7.8)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)